cdc/processor/sinkmanager/manager.go

```diff
@@ -185,7 +186,12 @@ func (m *SinkManager) Run(ctx context.Context, warnings ...chan<- error) (err er
 	}()
 
 	splitTxn := util.GetOrZero(m.changefeedInfo.Config.Sink.TxnAtomicity).ShouldSplitTxn()
-	enableOldValue := m.changefeedInfo.Config.EnableOldValue
```

Comment:
`enableOldValue` does not equal `ShouldSplitUpdate.` it also controls some behavior of the delete event.